### ISSUE TITLE
Stop recommending Anaconda

### ISSUE THREAD

**Issue by eerovaher** [Lund-Observatory-Teaching/lundpython#92]
[Currently we are recommending students to use Anaconda.](https://github.com/Lund-Observatory-Teaching/lundpython/blob/38e804ae85cfdde51146a470c603c51809ecb403/guide.md) Although Anaconda is (at the moment) free to use for educational purposes, [using Anaconda in large organizations normally requires buying a license.](https://www.anaconda.com/pricing/terms-of-service-faqs). We should prefer FOSS tools (to avoid vendor lock-in etc.). The least disruptive change would be to recommend Miniconda with the `conda-forge` channel. The best instructions for doing that that I know of are in the [`sunpy` documentation](https://docs.sunpy.org/en/stable/tutorial/installation.html#installing-miniforge). But next year we should consider recommending [`uv`](https://docs.astral.sh/uv/), if that project is in a good enough state.

**Comment by AlinderS** [comment]
I agree (at least in principle) with this. Another reason not to use Anaconda is that it is simply not very good at doing its job.
With consideration of my workload, my suggestion is to rewrite the `guide.md` page to recommend `uv` instead of Anaconda sometime between the end of the current course and the beginning of next year's course. We should also create and provide a requirements file so the students can run `uv pip sync requirements.txt` and have all the libraries they will need.
It is not at all clear that I will be actively involved with this course next year, so I can try to get this done during the fall.
